src/mongocrypt-marking.c

```diff
@@ -299,7 +299,7 @@ _fle2_placeholder_aes_ctr_encrypt (_mongocrypt_key_broker_t *kb,
    _mongocrypt_crypto_t *crypto = kb->crypt->crypto;
    _mongocrypt_buffer_t iv;
    const uint32_t cipherlen =
-      _mongocrypt_fle2_calculate_ciphertext_len (in->len);
+      _mongocrypt_fle2_calculate_ciphertext_len (in->len, status);
```

Comment:
Return an error if `_mongocrypt_fle2_calculate_ciphertext_len ` returns 0. Otherwise, an error status may be ignored by the caller.

Reply:
Change incoming